static/i18n/en/app.json

```diff
@@ -7159,7 +7159,7 @@
         "title": "Now game on!",
         "descr": "Answer 3 simple questions to avoid common misconceptions about your hardware wallet.",
         "buttons": {
-          "start": "Let’s take the quiz",
+          "start": "I'm ready",
```

Comment:
⚠️ Impacts V2

Reply:
It's in the v3 part of the json and the text in the design is actually different from v2, should I just create a new entry for this ?

Reply:
Oh my bad sorry I did not see that it was scoped inside the V3 part 🙇